google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -235,6 +269,21 @@ Builder setUniformBucketLevelAccessLockedTime(Long uniformBucketLevelAccessLocke
         return this;
       }
 
+      /**
+       * Sets the bucket's Public Access Prevention configuration. Currently supported options are
+       * 'unspecified' (default) or 'enforced'.
+       *
+       * @see <a
+       *     href="https://cloud.google.com/storage/docs/public-access-prevention">public-access-prevention</a>
+       */
+      public Builder setPublicAccessPrevention(PublicAccessPrevention publicAccessPrevention) {
+        this.publicAccessPrevention =
+            publicAccessPrevention == null
```

Comment:
It's possible for the value to be `null` if not selected in a GET request.

Reply:
> It's possible for the value to be `null` if not selected in a GET request.

yes, done. PTAL